starter-code/javascript/chronometer.js

```diff
@@ -8,19 +8,19 @@ class Chronometer {
   startClick(callback) {
     this.isRunning = true;
     this.intervalId = setInterval(() => {
-      this.currentTime += 1 ;
+      this.currentTime += 1;
       console.log(this.currentTime);
       callback();
-      }, 1000);
-    }
+    }, 1000);
+  }
 
   getMinutes() {
     let minutes = Math.trunc(this.currentTime / 60);
     return this.twoDigitsNumber(minutes);
   }
 
   getSeconds() {
-    let seconds = this.currentTime % 60 ;
+    let seconds = this.currentTime % 60;
     return this.twoDigitsNumber(seconds);
```

Comment:
this breaks the jasmine test aswell, you getSeconds should return a number, even though it is convenient for the lab to return a string,  the Chronometer class might be used in a different context than the DOM so it would still be good to return a number.